kolibri/core/logger/csv_export.py

```diff
@@ -154,7 +154,7 @@ def csv_file_generator(facility, log_type, filepath, overwrite=False):
     if sys.version_info[0] < 3:
         csv_file = io.open(filepath, "wb")
     else:
-        csv_file = io.open(filepath, "w", newline="")
+        csv_file = io.open(filepath, "w", newline="", encoding="utf-8")
```

Comment:
I see code similar to this in places like here

https://github.com/learningequality/kolibri/blob/ab3b75dc5f5e2fa4df303ce3dba086913618f727/kolibri/core/auth/csv_utils.py#L189-L193

does the `encoding` need to be set there too?

Reply:
You mean for the `sys.version_info[0] < 3` code path? No - I had originally added that and it broke stuff, I think it's not needed because it is opened in `'wb'` mode as opposed to just `'w'` - which again is due to weird differences between the csv module's behaviour in Python 2 vs 3.